php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -189,7 +189,7 @@ class NDB_BVL_Instrument extends NDB_Page
             && !file_exists($base."project/instruments/$instrument.linst")
             && $guarantee_exists==false
         ) {
-            return;
+            return null;
```

Comment:
function signature isn't nullable

Reply:
This function now does not have a return type as it must be able to return itself as well as child classes. The PHPDoc has been updated to show that it can return null in this case.